core/config/genesis/stage.json

```diff
@@ -1 +1,62 @@
-{}
+{
+  "genesis_time": "2024-08-09T00:00:00.0000Z",
+  "chain_id": "audius-testnet",
+  "initial_height": "0",
+  "consensus_params": {
+    "block": {
+      "max_bytes": "104857600",
+      "max_gas": "10000000"
+    },
+    "evidence": {
+      "max_age_num_blocks": "100000",
+      "max_age_duration": "172800000000000",
+      "max_bytes": "104857600"
+    },
+    "validator": {
+      "pub_key_types": [
+        "ed25519"
+      ]
+    },
+    "version": {
+      "app": "0"
+    },
+    "synchrony": {
+      "precision": "500000000",
+      "message_delay": "2000000000"
+    },
+    "feature": {
+      "vote_extensions_enable_height": "0",
+      "pbts_enable_height": "0"
+    }
+  },
+  "validators": [
+    {
+      "address": "0F4BE2AAA70E9570EEE3485D8FA54502CF1A9FC0",
+      "pub_key": {
+        "type": "tendermint/PubKeyEd25519",
+        "value": "IqMUsjq6IB+A6TZJnNRi195qrWptol9Tcr8Hpxx6zxE="
+      },
+      "power": "25",
+      "name": "foundation-discovery-2"
+    },
+    {
+      "address": "2F13439B2EE4C34BAFE643F89575F40B7863A079",
+      "pub_key": {
+        "type": "tendermint/PubKeyEd25519",
+        "value": "Ubt7ViBm+mG24G+bbG8NgL7NMuwZ/u6uyf9taTwgsEI="
+      },
+      "power": "25",
+      "name": "foundation-discovery-3"
+    },
+    {
+      "address": "C9B1ED3D3040E0C2AC70E3215F0EA9B16B401BCA",
+      "pub_key": {
+        "type": "tendermint/PubKeyEd25519",
+        "value": "mj5kYzxWEG7iAnHAiJoUiJoPIOItBOEdr6W5sGTf0iY="
+      },
+      "power": "25",
```

Comment:
do you mind changing the power to 10? i used 25 for local dev so the four nodes all add up to 100 but stage / prod don't have those same numbers